Formula/haskell-language-server.rb

```diff
@@ -0,0 +1,41 @@
+class HaskellLanguageServer < Formula
+  desc "Integration point for ghcide and haskell-ide-engine. One IDE to rule them all"
+  homepage "https://github.com/haskell/haskell-language-server"
+  url "https://github.com/haskell/haskell-language-server/archive/1.0.0.tar.gz"
+  sha256 "14e28d6621d029f027fae44bc4a4ef62c869dab24ff01b88a2e51e6679cbff6c"
+  license "Apache-2.0"
+  head "https://github.com/haskell/haskell-language-server.git"
+
+  # we need :github_latest here because otherwise livecheck picks up spurious non-release tags
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "cabal-install" => [:build, :test]
+  depends_on "ghc" => [:build, :test]
+
+  def install
+    system "cabal", "v2-update"
+    system "cabal", "v2-install", *std_cabal_v2_args
+  end
+
+  def caveats
+    versioned_ghc = "ghc #{Formula["ghc"].version}"
+
+    <<~EOS
+      #{name} is built for #{versioned_ghc}. You will need to provide
+      your own #{versioned_ghc} or install one with
+        brew install ghc
+
+      See: https://github.com/haskell/haskell-language-server#prerequisites
```

Comment:
```suggestion
```
We try to keep caveats to the minimum that's specific to Homebrew. This is documentation that would be relevant even if you weren't using Homebrew, so I don't think we need this.

Reply:
Fair enough.